fastlane/lib/fastlane/actions/device_grid/AppetizeFastfile

```diff
@@ -0,0 +1,22 @@
+desc "Generate and upload an .app to appetize.io"
+desc "Import this lane and call it from Danger"
+
+private_lane :build_and_upload_appetize do |options|
+  tmp_path = "/tmp/fl_build"
+
+  xcodebuild_configs = options[:xcodebuild]
+  xcodebuild_configs[:sdk] = "iphonesimulator"
+  xcodebuild_configs[:derivedDataPath] = tmp_path
+  
+  xcodebuild(xcodebuild_configs)
+
+  app_path = Dir[File.join(tmp_path, "**", "*.app")].last
```

Comment:
1. if you build mutiple apps, won't that get you into issues here (i.e. it won't necessarily find the right one ?). Maybe randomize the `tmp_path` ?

2. I remember from the issue tracker that someone (was it @jcampbell05 ?) had problems with a bug in xcode/cocoa resulting in packaging of multiple apps being packaged (e.g. watch apps as well). Could  this also cause issues ? (though someone can fork and patch this Fastfile then).